internal/quickfort/query.lua

```diff
@@ -16,6 +16,7 @@ local log = quickfort_common.log
 
 local function is_queryable_tile(pos)
     local flags, occupancy = dfhack.maps.getTileFlags(pos)
+    if not flags then return false end
```

Comment:
protect against the tile being out of map bounds (where `getTileFlags` returns `nil`)